packages/wallets/src/features/cashier/modules/DepositLocked/__tests__/DepositLocked.spec.tsx

```diff
@@ -41,17 +41,17 @@ const mockStatusData = {
 };
 
 describe('DepositLocked', () => {
-    afterEach(() => {
+    beforeEach(() => {
         jest.clearAllMocks();
     });
 
     it('should render loader when no account status data', () => {
-        (useActiveWalletAccount as jest.Mock).mockReturnValueOnce({});
-        (useSettings as jest.Mock).mockReturnValueOnce({ data: mockSettingsData });
-        (useWebsiteStatus as jest.Mock).mockReturnValueOnce({ data: mockWebsiteStatusData });
-        (useAuthentication as jest.Mock).mockReturnValueOnce({ data: mockAuthenticationData });
-        (useCashierValidation as jest.Mock).mockReturnValueOnce({ data: mockCashierValidationData });
-        (useAccountStatus as jest.Mock).mockReturnValueOnce({ data: null });
+        (useActiveWalletAccount as jest.Mock).mockReturnValue({});
+        (useSettings as jest.Mock).mockReturnValue({ data: mockSettingsData });
+        (useWebsiteStatus as jest.Mock).mockReturnValue({ data: mockWebsiteStatusData });
+        (useAuthentication as jest.Mock).mockReturnValue({ data: mockAuthenticationData });
+        (useCashierValidation as jest.Mock).mockReturnValue({ data: mockCashierValidationData });
+        (useAccountStatus as jest.Mock).mockReturnValue({ data: null });
```

Comment:
@lubega-deriv, great job, but it's probably best to keep these for a different card, as this increases work for reviewers. I think it's best to stick to the card's scope.

```suggestion
        (useActiveWalletAccount as jest.Mock).mockReturnValue({});
        (useSettings as jest.Mock).mockReturnValue({ data: mockSettingsData });
        (useWebsiteStatus as jest.Mock).mockReturnValue({ data: mockWebsiteStatusData });
        (useAuthentication as jest.Mock).mockReturnValue({ data: mockAuthenticationData });
        (useCashierValidation as jest.Mock).mockReturnValue({ data: mockCashierValidationData });
        (useAccountStatus as jest.Mock).mockReturnValue({ data: null });
```

Reply:
@jim-deriv this change is needed as using mockReturnValueOnce was causing the mocks to leak to other test cases and causing them to fail 😅

Reply:
Alright, but even the ones in files such as this `packages/wallets/src/features/cfd/modals/MT5AccountAdded/__tests__/MT5AccountAdded.spec.tsx`? 🤔

Reply:
ah for that i was just adding some missing test cases since the component is under the scope of the refactor 👀